### PR TITLE
Make README friendlier

### PR DESCRIPTION
**Pull request by catenoid**
Attempt to move the more technical / development-oriented content from the README to the wiki, and add information on how to run the docker image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,106 +2,36 @@
 
 # ApiCenter
 
-ApiCenter is a repository for your OpenAPI and GraphQL specifications. Through the web interface you can upload specifications, interact with them through the [SwaggerUI](https://swagger.io/tools/swagger-ui/) / [GraphiQL](https://github.com/graphql/graphiql), and search their contents.
+ApiCenter is a repository for OpenAPI and GraphQL specifications. Through the web interface you can upload & download specifications, interact with them through the [SwaggerUI](https://swagger.io/tools/swagger-ui/) / [GraphiQL](https://github.com/graphql/graphiql), and search their contents.
```
Comment (fdw) [inline README.md]: "and"

File: README.md [hunk 2]
```diff
@@ -2,106 +2,36 @@
...
-#### Combined
-From the project root directory, run:
-```
-./gradlew :monoBuild
-```
-The `:monoBuild` task is composed of three subtasks, which can be independently invoked if needed:
-- `:frontend:buildClientDev` runs Angular's build command, producing the output files in the default location `frontend/dist`.
-- `:frontend:copyFrontendFiles` copies these files into a directory in `backend` from which Spring will serve static resources.
-- `:backend:bootRun` will compile and run the backend.
+* In a browser, go to `<that-IP-Address>:8080`
```
Comment (fdw) [inline README.md]: Isn't there something possible with `localhost:8080`?

File: README.md [hunk 3]
```diff
@@ -2,106 +2,36 @@
...
 
-The service is available from `localhost:8080` (Spring boot's default port).
+By default, user authentication is disabled. Login succeeds regardless of what password is entered.
 
-#### Separate
+## How to test
```
Comment (fdw) [inline README.md]: I'd rather move this to the wiki

File: README.md [hunk 4]
```diff
@@ -2,106 +2,36 @@
 
 # ApiCenter
 
-ApiCenter is a repository for your OpenAPI and GraphQL specifications. Through the web interface you can upload specifications, interact with them through the [SwaggerUI](https://swagger.io/tools/swagger-ui/) / [GraphiQL](https://github.com/graphql/graphiql), and search their contents.
+ApiCenter is a repository for OpenAPI and GraphQL specifications. Through the web interface you can upload & download specifications, interact with them through the [SwaggerUI](https://swagger.io/tools/swagger-ui/) / [GraphiQL](https://github.com/graphql/graphiql), and search their contents.
 
-## How to use
+More information and troubleshooting is on the [wiki](https://github.com/TNG/ApiCenter/wiki).
```
Comment (fdw) [inline README.md]: Some screenshots, maybe?

Comment (catenoid) [reply README.md]: I decided against this, thinking that for the README to render locally, these images would have to be committed as part of the project. And the screenshots aren't really static assets, they'd need updating in response to cosmetic UI changes, which in my mind would just create noise in the commit history. It's much simpler to manage screenshots / related media on the wiki, as Github can host everything. Does that seem reasonable? I am open to counter arguments.

Comment (fdw) [reply README.md]: Your argument definitely makes sense. My only counterpoint is that the readme is the first things people see, as it is shown on the main Github page. If we don't capture interest here, they won't click through to the wiki.

Also, the static web page would require screenshots, too, and it is usually also kept in the repo (see Github pages).

**Pull request merged**